shepherd_bms/src/compute.cpp

```diff
@@ -34,7 +33,7 @@ FaultStatus_t ComputeInterface::sendChargingMessage(uint16_t voltageToSet, uint1
         } cfg;
     } chargerMsg;
 
-    if (!isChargingEnabled)
+    if (!is_charging_enabled)
```

Comment:
Is snake_case what we should be using? Don't we use camelCase?

Reply:
should rework wording on confl but nah this is correct, snake_case for variables camelCase for functions